modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,340 @@
+                "elasticloadbalancing:AddListenerCertificates",
+                "elasticloadbalancing:RemoveListenerCertificates",
+                "elasticloadbalancing:ModifyRule"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+----
++
+.. Create an AWS IAM role and assign the AWS IAM policy displayed in step 2a to that role. For more information on creating AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html[Creating IAM roles]. For more information on adding AWS IAM policies to AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html[Adding and removing IAM identity permissions].
+
+.. Add the following trust policy to the newly-created AWS IAM role. For more information on adding trust policies to AWS IAM roles, see link:https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/[How to use trust policies with IAM roles].
+
++
+--
+[source,json]
+----
+"{
+    ""Version"": ""2012-10-17"",
+    ""Statement"": [
+        {
+            ""Effect"": ""Allow"",
+            ""Principal"": {
+                ""Federated"": ""arn:aws:iam::{AWS_AccountNo}: <1>
+
+                {Cluster_OIDC_Endpoint}"" <2>
+            },
+            ""Action"": ""sts:AssumeRoleWithWebIdentity"",
+            ""Condition"": {
+                ""StringEquals"": {
+                    ""{Cluster's OIDC Endpoint}:sub"": [
+                        ""system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-operator-controller-manager"",
+                        ""system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-controller-cluster""
+                    ]
+                }
+            }
+        }
+    ]
+}"
+----
+<1> Replace '{AWS_AccountNo}' with your AWS account number.
+<2> Replace '{Cluster_OIDC_Endpoint}' with the OIDC DNS specified in
+step 1b.
+
+[NOTE]
+====
+Do not include the 'https' portion of the OIDC DNS url. Only the alphanumeric information that proceeds the '.net/' within the URL is necessary for this task.
+====
+--
+. Add the tags necessary for subnet discovery:
+
+.. Add the following {Key: Value} tag to the VPC hosting the {product-title} cluster, replacing '{Cluster Infra ID}' with the Infra ID specified in step 1a:
+
+
+* kubernetes.io/cluster/${Cluster Infra ID}:owned
+
+.. Add the following Elastic Load Balancer (ELB) {Key: Value} tags to private subnets and (optional) public subnets.
+
+* Private Subnets: kubernetes.io/role/elb:1
+* Public subnets and kubernetes.io/role/interna-elb:1
+
+[NOTE]
+====
+Internet-facing and internal load balancers will be created within the  Availability Zones (AZ) of which these subnets belong.
+====
+
+For more information on adding tags to AWS resources (including VPCs and subnets), see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tag your Amazon EC2 resources].
```

Comment:
This whole section comes out broken in the overall page, please check.
The numbered lists restart from 1. There's perhaps a title missing??

Reply:
> This whole section comes out broken in the overall page, please check. The numbered lists restart from 1. There's perhaps a title missing??

Ack. Yes, there is an issue with the numbering I am trying to resolve. That whole section needs to be worked on (hadn't started on that section yet).

Reply:
> This whole section comes out broken in the overall page, please check. The numbered lists restart from 1. There's perhaps a title missing??

The numbering issue was resolved.